ibm_mq/datadog_checks/ibm_mq/ibm_mq.py

```diff
@@ -129,3 +143,26 @@ def send_metrics_from_properties(self, properties, metrics_map, prefix, tags):
                     )
                     return
                 self.send_metric(metric_type, metric_full_name, metric_value, new_tags)
+
+    def check_queue_manager_process(self):
+        if self.queue_manager_process_pattern is None:
+            return True
+
+        return self.process_matcher.check_condition(self.check_id, self.queue_manager_process_pattern, self.log)
+
+    def reset_queue_manager_process_match(self):
+        if self.queue_manager_process_pattern is not None:
+            self.log.debug('Resetting queue manager process match')
+            return self.process_matcher.remove(self.check_id)
+
+    def parse_config(self):
+        pattern = self.instance.get('queue_manager_process', self.init_config.get('queue_manager_process', ''))
+        if pattern:
+            if PY2:
+                raise ConfigurationError('The `queue_manager_process` option is only supported on Agent 7')
+
+            pattern = pattern.replace('<queue_manager>', re.escape(self.instance['queue_manager']))
+            self.queue_manager_process_pattern = re.compile(pattern)
+
+    def cancel(self):
```

Comment:
What is this being used for?

Reply:
https://github.com/DataDog/integrations-core/blob/7.40.x/datadog_checks_base/datadog_checks/base/checks/base.py#L1075-L1082

Reply:
Here is where `@Override` annotation comes handy in Java. Since we don't use this much I think I'd be nice to have a comment for this method